src/Common/Infrastructure/Service/LicenseFeeReader.php

```diff
@@ -37,6 +38,27 @@ public function __construct(LicenseVault $licenseVault)
         $this->licenseVault = $licenseVault;
     }
 
+    public function getAddress(): AccountId
+    {
+        $value = apcu_fetch(Config::LICENCE_ACCOUNT);
+
+        if ($value) {
+            return new AccountId($value);
+        }
+
+        try {
+            $license = $this->licenseVault->read();
+        } catch (RuntimeException $exception) {
+            return new AccountId(Config::getLicenceAccount());
```

Comment:
Now Config::getLicenseAccount may throw a ```Illuminate\Database\Eloquent\ModelNotFoundException```. Don't we want to handle it somehow?

Reply:
Now it's not only LicenseFeeReader, maybe we should change the name into - ```LicenseReader``` or ```LocalLicenseReader``` ?